food-truck-frontend/src/components/ReviewsList.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {Component} from 'react';
+import api from "../util/api";
+
+const reviewsJSON = [{
+    "id": "",
+    "userId": "",
+    "truck": {"id": "",
+        "userId": "",
+        "name": "",
+        "menu": "",
+        "textMenu": "",
+        "priceRating": "",
+        "description": "",
+        "schedule": "",
+        "foodCategory": ""},
+    "starRating": "",
+    "costRating": "",
+    "reviewText": "",
+    "day_time": ""
+}];
+
+type Props = {
+    username: string
+}
+
+type State = {
+    data : {
+        "id": string,
+        "userId": string,
+        "truck": {"id": string,
+            "userId": string,
+            "name": string,
+            "menu": string,
+            "textMenu": string,
+            "priceRating": string,
+            "description": string,
+            "schedule": string,
+            "foodCategory": string},
+        "starRating": string,
+        "costRating": string,
+        "reviewText": string,
+        "day_time": string
+    }[],
+}
+
+
+
+
+
+class ReviewsList extends React.Component<Props, State>{
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {data: reviewsJSON};
+    }
+
+
+    componentDidMount(){
+        api.get('/user/reviews?username=' + this.props.username).then((response) => {
```

Comment:
Micah mentioned this somewhere else, but you can also use the `${...}` syntax here